scratchml/models/multilayer_perceptron.py

```diff
@@ -333,6 +350,12 @@ def fit(self, X: np.ndarray, y: np.ndarray) -> None:
                         loss = binary_cross_entropy(batch_y, y_hat, derivative=True)
                     elif self.loss_function == "cross_entropy":
                         loss = cross_entropy(batch_y, y_hat, derivative=True)
+                elif isinstance(self, MLPRegressor):
+                    if self.loss_function == "mse":
+                        loss = 2 * (y_hat - batch_y) / len(batch_y)  # Derivative of MSE w.r.t predictions
```

Comment:
Please, update that line to use the "mean_squared_function" defined in line 4 on the metrics.py file. For example:

```python
loss = 2 * mean_squared_error(batch_y, y_hat, derivative=True) / len(batch_y)
```

Reply:
Done!